gamemodes/PPC_Trucking.pwn

```diff
@@ -409,6 +409,7 @@ public OnPlayerDisconnect(playerid, reason)
 		APlayerData[playerid][RentedVehicleID] = 0;
 	}
 
+   SendDeathMessage(INVALID_PLAYER_ID, playerid, 201); // 201 = DEATHMESSAGE_DISCONNECT
```

Comment:
Please undo this change as we already have chat messages that inform about players who join / leave the server. If you want file an issue regarding the usage of `SendDeathMessage` in general.

Reply:
Opened issue for this, see #87